frontend/src/features/admin-form/template/UseTemplateModal/UseTemplateWizardProvider.tsx

```diff
@@ -22,6 +22,8 @@ export const useUseTemplateWizardContext = (
       /* enabled= */ !!formId,
     )
 
+  const hasSingpass = !!templateFormData?.spcpSession
```

Comment:
nit: `has*` is quite non-standard. would using `is*` be better instead? for e.g. `isSpcpEnabled` or something like that

can use this naming throughout the PR